### PR TITLE
Autcomplete file paths

### PR DESCRIPTION
**Pull request by satwikkansal**
Requires https://gitlab.com/coala/coala-utils/merge_requests/47 to be merged first.

### CODE REVIEW

File: coala_quickstart/coala_quickstart.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 from coalib.output.printers.LogPrinter import LogPrinter
 
+from coala_utils.FilePathCompleter import FilePathCompleter
```
Comment (Nosferatul) [inline coala_quickstart/coala_quickstart.py]: i think the problem is that it doesn't see this file.

Comment (satwikkansal) [inline coala_quickstart/coala_quickstart.py]: Yep, the comment https://github.com/coala/coala-quickstart/pull/172#issue-249149958 mentions it :)

**Comment by adtac** [comment]
ack https://github.com/coala/coala-quickstart/pull/172/commits/313a9816219361e0fb4e5cffe5915de475bcf8a9

**Comment by adtac** [comment]
unack https://github.com/coala/coala-quickstart/pull/172/commits/63fe4e872260870e66b895e071bbb34a650c9e7c

**Comment by satwikkansal** [comment]
reack 1826d98

**Comment by Nosferatul** [comment]
I rebuilded appveyor

**Comment by satwikkansal** [comment]
@Nosferatul it's failing due to https://gitlab.com/coala/coala-utils/issues/40 and the fix is https://gitlab.com/coala/coala-utils/merge_requests/49 :)

**Comment by satwikkansal** [comment]
reack 462beb3 eb5053f

**Comment by adtac** [comment]
@rultor merge

**Pull request merged**